.github/workflows/release_beta.yml

```diff
@@ -0,0 +1,109 @@
+name: Release
```

Comment:
Is there any reason to add a new `-beta` workflow, rather than just replacing the old one?

These should be versioned so a release from the v2 branch should use the workflow present on that branch.

Reply:
yes, true, changed.